doc/efun/configure_driver

```diff
@@ -118,6 +118,28 @@ DESCRIPTION
            the swap file as small as possible.
            (Same as the --swap-compact command line switch.)
 
+        <what> == DC_SWAP_TIME
+           Sets the time until the program of an unused object is swapped out
+           (if possible).
+           <data> is an integer and measured in seconds.
+           (Same as the --swap-time command line switch.)
+
+        <what> == DC_SWAP_VAR_TIME
+           Sets the time until the variables of an unused object are swapped
+           out.
+           <data> is an integer and measured in seconds.
+           (Same as the --swap-variablescommand line switch.)
+
+        <what> == DC_CLEANUP_TIME
+           Sets the time until the cleanup hook is called in unused objects.
+           <data> is an integer and measured in seconds.
+           (Same as the --cleanup-time command line switch.)
+
+        <what> == DC_RESET_TIME
+           Sets the default time until the reset hook is called in objects.
+           <data> is an integer and measured in seconds.
+           (Same as the --reset-time command line switch.)
```

Comment:
Vielleicht so ein Hinweis a la: The change will take effect for each object after its next reset.